Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -465,6 +477,7 @@ void WebSocketHLEFuncRename(DebuggerRequest &req) {
 // Parameters:
 //  - address: unsigned integer address for the start of the range.
 //  - size: unsigned integer size in bytes for scan.
+//  - recreate: optional bool indicating whether functions that intersect or inside lie inside the range must be removed before scanning
```

Comment:
I'd rather call this `clear` or something.  For example, if the detection code does not detect any function symbols in the range, there's certainly no guarantee it would just recreate the old ones.  The name and description implies it does something different than it does.

It'd be more direct if it was `clear` or `forget` or `remove`, and described as an optional bool which indicates whether to remove all functions in the range before scanning.

-[Unknown]